build.gradle

```diff
@@ -131,8 +131,8 @@ ext {
     // NB: due to version.json generation by parsing this file, the next line must not have any if/then/else logic
     neo4jVersion = "5.0.0"
     // This should be removed when the version is stable and released
-    neo4jInDevVersion = (System.env.CI != null) ? neo4jVersion + "-dev" : neo4jVersion + "-SNAPSHOT"
+    // neo4jInDevVersion = (System.env.CI != null) ? neo4jVersion + "-dev" : neo4jVersion + "-SNAPSHOT"
```

Comment:
Do we still want this commented out, or will it only cause confusion?

Reply:
This is always going to be needed in the dev branch, so I would prefer to leave it commented